device_plugins/deploy_qat.md

```diff
@@ -76,6 +76,38 @@ Users can use the steps below to customize the QAT resource configuration:
 
 2. Create QAT device plugin CR with -provisioning-config set as the name of the ConfigMap (created in step 1) in the qat_device_plugin.yaml file or set ConfigMap name in the provisioning-config option from web console. 
 
+# Multiple Custom Resources
+
+The [feature](https://github.com/intel/intel-device-plugins-for-kubernetes/tree/main/cmd/operator#multiple-custom-resources) can be used with a `nodeSelector` label representing the capabilities supported on the node. Multiple custom resources on nodes can be supported with different QAT capabilities.
+
+* For example, to assign the capability `sym` for a node: create the label on the node
```

Comment:
I think this example misses the part that how *multiple* custom resources are deployed?